deliverables/ruby_monk_problems/palindrome.rb

```diff
@@ -1,3 +1,12 @@
 def palindrome?(sentence)
-  # Write your code here
-end
+  #Strip white space
+  sentence = sentence.gsub!(/\s+/, "")
+  #Make all lower case
+  sentence = sentence.downcase
+  #Reverse sentence
+  reverse_sentence = sentence.reverse
+  #Compare reverse sentence to original
+  true if reverse_sentence == sentence
+  #Return true
+  
```

Comment:
Ok so first off good job :+1:  I like that you tried to actually code this.. Couple issues is that `sentence` gsub bang on line 3 will return nil. 

The solution is not valid if I pass in a single word like 'mom'